PLEP-0002.md

```diff
@@ -0,0 +1,123 @@
+# PLEP-0002 -- Coordinating Committee
+
+| PLEP              | 2                                 |
+|-------------------|-----------------------------------|
+| title             | Coordinating Committee            |
+| author(s)         | Nick Murphy                       |
+| contact email     | namurphy@cfa.harvard.edu          |
+| date created      | 2017-11-15                        |
+| date last revised | 2017-11-15                        |
+| type              | process                           |
+| status            | in preparation                    |
+
+# Abstract
+
+PlasmaPy's **Coordinating Committee** (CC) shall oversee the PlasmaPy
+project and code development.
+
+# Detailed Description
+
+## Membership of Coordinating Committee
+
+### Limits on number of members
+
+The CC shall consist of at least 4 members and no more than 10
+members.  Within these limits, the CC may increase or decrease the
+number of directors serving on the CC, including for the purposes of
+staggering the terms of members.
+
+### Term of office
+
+All members shall be elected to serve terms of up to two years.
+However, the CC may extend the term until a successor has been
+elected.  A member may be re-elected at the end of their term.  The
+term of a CC member shall begin on January 1 of the first year in
+office and end on December 31 of the second year in office, unless the
+CC chooses different dates.  The terms should be staggered so that
+roughly half of the terms expire each year.
```

Comment:
Instead of having staggered two year terms, each member of the Coordinating Committee could have a term length of a different prime number of years so that we can protect ourselves from predators using the same strategy as [cicadas](https://en.wikipedia.org/wiki/Periodical_cicadas#Predator_satiation_survival_strategy).  🐞 🐞 🐞 🐞 🐞 🐞 🐞 🐞 🐞 🐞 🐞 🐞 🐞 🐞 🐞